### PR TITLE
Fix reported duration of check

### PR DESCRIPTION
**Pull request by jsoref**
`... Successful in -1m`

probot/probot#803

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -12590,7 +12569,7 @@
     },
     "probot-config": {
       "version": "git+https://github.com/bobvanderlinden/probot-config.git#9825185ee0a1698120193ad34e6832bc43f90e18",
-      "from": "git+https://github.com/bobvanderlinden/probot-config.git#9825185ee0a1698120193ad34e6832bc43f90e18",
+      "from": "git+https://github.com/bobvanderlinden/probot-config.git#pr-probot-7.4.0",
```
Comment (bobvanderlinden) [inline package-lock.json]: I'm not sure whether this is intended, but this change seems unrelated to the PR?

Comment (jsoref) [reply package-lock.json]: Err, no idea how that happened. Sorry.

Comment (jsoref) [reply package-lock.json]: My guess is that came from me running `npm it` or similar (it's really great that projects have such a verb, it's such a pain that the verb causes churn).

**Comment by bobvanderlinden** [comment]
This looks good 👍 Thank you!

**Comment by bobvanderlinden** [comment]
Sorry for the delay. Finally figured out Travis build problems I was running into on master. (https://travis-ci.community/t/cached-builds-result-in-module-not-found/1861) They should be solved now.
